From 67f044f74a49ecf4c5b44e08de0d4871616752de Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Mon, 17 Jul 2017 11:22:19 -0500
Subject: [PATCH 237/278] drivers/misc: Update the Axxia Fault Handler

Mask exceptions by default on 5500.  Exceptions are
not masked by default on 5600 or 6700.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 drivers/misc/axxia-fault.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/misc/axxia-fault.c b/drivers/misc/axxia-fault.c
index e9ab5da..5918f54 100644
--- a/drivers/misc/axxia-fault.c
+++ b/drivers/misc/axxia-fault.c
@@ -157,6 +157,7 @@
 #if defined(ARM)
 	if (of_find_compatible_node(NULL, NULL, "lsi,axm5500") ||
 	    of_find_compatible_node(NULL, NULL, "lsi,axm5516")) {
+		mask_aborts = 1;
 		hook_fault_code(0x11, axxia_x7_fault, SIGBUS, 0,
 				"asynchronous external abort");
 		pr_debug("Set up fault handler for Axxia 5500\n");
-- 
1.8.5.2.233.g932f7e4

